sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.NUnit/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 # Release History
 
-## 1.0.0-beta.4 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 1.0.0-beta.4 (2024-12-13)
 
 ### Bugs Fixed
 
-### Other Changes
+- Fixed a bug where the API call was failing due to the character limit in the test suite ID.
```

Comment:
Fixed a bug where upload results were incomplete for large size classname suites